app-crypt/tpm2-tools/tpm2-tools-5.2.ebuild

```diff
@@ -0,0 +1,71 @@
+# Copyright 1999-2021 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+PYTHON_COMPAT=( python3_{8,9} )
+inherit autotools bash-completion-r1 python-single-r1
+
+DESCRIPTION="Tools for the TPM 2.0 TSS"
+HOMEPAGE="https://github.com/tpm2-software/tpm2-tools"
+SRC_URI="https://github.com/tpm2-software/${PN}/releases/download/${PV}/${P}.tar.gz"
+
+LICENSE="BSD"
+SLOT="0"
+KEYWORDS="~amd64 ~x86"
+IUSE="+fapi test"
+
+RDEPEND=">=app-crypt/tpm2-tss-3.0.1[fapi?]
+	dev-libs/openssl:=
+	net-misc/curl
+	sys-libs/efivar:=
+	${PYTHON_DEPS}"
+DEPEND="${RDEPEND}
+	test? (
+		app-crypt/swtpm
+		app-crypt/tpm2-abrmd
+		app-editors/vim-core
+		dev-tcltk/expect
+		dev-util/cmocka
+		dev-python/pyyaml
+	)"
+BDEPEND="virtual/pkgconfig
+	sys-devel/autoconf-archive"
+
+RESTRICT="!test? ( test )"
+REQUIRED_USE=" ${PYTHON_REQUIRED_USE} "
```

Comment:
Just style, but these are more commonly placed after IUSE given what they use is related and it's nice to be grouped together, e.g.
```
KEYWORDS=
IUSE=
REQUIRED_USE=
RESTRICT=
```
Also, no need for the spaces in REQUIRED_USE

Reply:
Moved